File: src/main/java/com/battletrident/commands/command/BT.java
package com.battletrident.commands.command;

import com.battletrident.guis.adminGUI.AdminGUI;
import io.papermc.paper.command.brigadier.BasicCommand;
import io.papermc.paper.command.brigadier.CommandSourceStack;
import org.bukkit.entity.Player;

public class BT implements BasicCommand {
	@Override
	public void execute(CommandSourceStack stack, String[] args) {
		Player player = (Player)stack.getSender();
		if (player.isOp() == false) return;

		player.openInventory(new AdminGUI().getInventory());
	}
}

File: src/main/kotlin/com/BattleTrident/commands/command/BT.kt
package com.BattleTrident.commands.command

import io.papermc.paper.command.brigadier.BasicCommand
import io.papermc.paper.command.brigadier.CommandSourceStack
import com.BattleTrident.guis.adminGUI.AdminGUI
import org.bukkit.entity.Player

class BT : BasicCommand {
	override fun execute(stack: CommandSourceStack, args: Array<String>) {
		val sender = stack.sender

		if (
			sender is Player &&
			sender.isOp
		) {
			sender.openInventory(AdminGUI().inventory)
		}
	}
}